src/util/util_bit.h

```diff
@@ -1,16 +1,29 @@
 #pragma once
 
-#ifndef _MSC_VER
-#if defined(__WINE__) && defined(__clang__)
-#pragma push_macro("_WIN32")
-#undef _WIN32
-#endif
-#include <x86intrin.h>
-#if defined(__WINE__) && defined(__clang__)
-#pragma pop_macro("_WIN32")
-#endif
+#if defined(__x86_64__) || defined(_M_X64) || defined(__i386__) || defined(_M_IX86)
+  #define DXVK_ARCH_X86
+  #if defined(__x86_64__) || defined(_M_X64)
+    #define DXVK_ARCH_X86_64
```

Comment:
If I'm reading the code right, why are we having 2 defines of 64-bits: DXVK_ARCH_x86_64 and DXVK_ARCH_x86 ?

Reply:
DXVK_ARCH_x86 is defined for 64 bit and 32 bit.. DXVK_ARCH_x86_64 is defined for 64 bit only.